internal/server/grpc.go

```diff
@@ -28,5 +28,6 @@ func NewGRPCServer(c *conf.Bootstrap, verifier *service.VerifierService, logger
 	}
 	srv := grpc.NewServer(opts...)
 	v1.RegisterVerifierServer(srv, verifier)
+	v1.RegisterTransactionServer(srv, transaction)
```

Comment:
fyi verifier is not a public server. It won't receive the request to CRUD the tx. The flow is:
1. The verifier is up and running to listen on-chain events.
2. Once it receives the attestation posting events, it fetch the attestation, verify then post the results on-chain.
The only thing we need the grpc server is the admin endpoint, which right now can only support some queries and also health check.